build-logic-settings/dependency-plugin/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+plugins {
+    `kotlin-dsl`
+}
+
+dependencies {
+    implementation("com.gradle.enterprise:com.gradle.enterprise.gradle.plugin:3.6.3")
```

Comment:
Is it needed? Didn't find usages in plugins

Reply:
it is used right in this module

```
plugins {
    id("com.gradle.enterprise")
}
```

scheme is:
-you specify dependency of convention plugin module (to add it to it's classpath to be able to use it's classes)
-you apply plugin without version, because it's already in classpath (it's a rule, you can' specify versions in convention plugin)

Reply:
I see how dependencies are used in conventions plugins.
The question is only about `enterprise.gradle.plugin`.

I found `id("com.gradle.enterprise")` only in `scan-plugin` module, but not in `dependency-plugin` module. This is the point.

Reply:
ah, sorry. now i see

Reply:
https://github.com/avito-tech/avito-android/pull/1139